projects/graph/src/draw.py

```diff
@@ -1,14 +1,137 @@
 """
```

Comment:
Consider adding a *shebang* at the top of your Python files and leaving more """Docstring""" comments to help with practicing your in code documentation. They're can be very useful and important when you get a career.